apps/mobile/src/features/send/send-form/hooks/use-send-form-stx.tsx

```diff
@@ -43,27 +42,32 @@ export function useSendFormStx() {
   const { displayToast } = useToastContext();
   const generateTx = useGenerateStxTokenTransferUnsignedTransaction(payer, publicKey);
 
-  function handleNonCompliantAddress(address: string) {
-    void analytics?.track('non_compliant_entity_detected', { address });
-    // eslint-disable-next-line lingui/no-unlocalized-strings
+  function handleNonCompliantAddress() {
     throw new StacksError('NonCompliantAddress');
   }
   return {
     onGoBack() {
       navigation.navigate('send-select-asset', { account });
     },
-    async onInitSendTransfer(data: SendFormStxContext, values: SendFormStxSchema) {
+    async onInitSendTransfer({ availableBalance }: SendFormStxContext, values: SendFormStxSchema) {
+      const parsedValues = parseSendFormValues(values);
+      const { amount, fee, recipient } = parsedValues;
       try {
-        const parsedValues = parseSendFormValues(values);
-        const { amount, recipient } = parsedValues;
         // TODO LEA-1852 - move to form schema validation
         // 1. validate precision
         if (!isValidPrecision(+values.amount, STX_DECIMALS)) {
-          // eslint-disable-next-line lingui/no-unlocalized-strings
-          throw new StacksError('InvalidPrecision');
+          // throw new StacksError('InvalidPrecision');
+          throw new Error('Pete');
```

Comment:
Do we really want to throw you as an error @pete-watters ? Seems a bit tragic.

Reply:
haha sorry, my bad!